packages/fluentui/react/src/themes/teams/components/Avatar/avatarStyles.ts

```diff
@@ -28,27 +28,34 @@ const avatarStyles: ComponentSlotStylesPrepared<AvatarStylesProps, AvatarVariabl
       width: sizeInRem
     };
   },
-  image: ({ variables: v }): ICSSInJSStyle => ({
+  image: ({ props: p, variables: v }): ICSSInJSStyle => ({
     borderColor: v.avatarBorderColor,
     borderStyle: 'solid',
     borderWidth: v.avatarBorderWidth,
 
     height: '100%',
     objectFit: 'cover',
     verticalAlign: 'top',
-    width: '100%'
+    width: '100%',
+
+    ...(p.square && {
+      borderRadius: v.avatarBorderRadius
+    })
   }),
-  label: ({ props: { size } }): ICSSInJSStyle => {
-    const sizeInRem = pxToRem(sizeToPxValue[size]);
+  label: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const sizeInRem = pxToRem(sizeToPxValue[p.size]);
     return {
       display: 'inline-block',
       width: sizeInRem,
       height: sizeInRem,
       lineHeight: sizeInRem,
-      fontSize: pxToRem(sizeToPxValue[size] / 2.333),
+      fontSize: pxToRem(sizeToPxValue[p.size] / 2.333),
       verticalAlign: 'top',
       textAlign: 'center',
-      padding: '0px'
+      padding: '0px',
+      ...(p.square && {
+        borderRadius: v.avatarBorderRadius
```

Comment:
```suggestion
        borderRadius: v.squareBorderRadius
```

Should not contain prop's name?